.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
+- [ ] Code reviewed
+- [ ] Code tested
```

Comment:
I guess the writer of the code can test, and the reviewer can test. Who is going to check the box?